### PR TITLE
PeerDb changes, removed static methods and initialization

### PR DESCRIPTION
**Pull request by yuriy-larin-firstbridge**
Removed static calls to PeerDb component

### CODE REVIEW

**Review by alukin — APPROVED** [review]

**Review by alzinchenko — APPROVED** [review]

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java [hunk 1]
```diff
@@ -16,40 +17,48 @@
...
-    public PeerLoaderThread(List<String> defaultPeers, List<Future<String>> unresolvedPeers, TimeService timeService, PeersService peersService) {
+    public PeerLoaderThread(List<String> defaultPeers,
+                            List<Future<String>> unresolvedPeers,
+                            TimeService timeService,
+                            PeersService peersService) {
         this.defaultPeers = defaultPeers;
         this.unresolvedPeers = unresolvedPeers;
         this.timeService = timeService;
         this.peersService = peersService;
+        this.peerDb = peersService.getPeerDb();
```
Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java:38]: Maybe better to inject PeerDb?

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java [hunk 2]
```diff
@@ -16,40 +17,48 @@
...
+            log.error(error);
+            throw new RuntimeException(error);
         }
         final int now = timeService.getEpochTime();
         peersService.wellKnownPeers.forEach((address) -> {
             PeerAddress pa = new PeerAddress(address);
             entries.add(new PeerDb.Entry(pa.getAddrWithPort(), 0, now));
         });
-        if (peersService.usePeersDb) {
+        if (PeersService.usePeersDb) {
```
Comment (AndrewBoyarsky) [inline apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java:55]: I guess, that all those options from PeersService (userPeersDb, savePeers, etc) should not be static

**Review by oleksiylukin — APPROVED** [review]

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java (hunk 1 above)
Comment (yuriy-larin-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java:38]: That was first variant, but I changed that later to less params in construcor.

File: apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java (hunk 2 above)
Comment (yuriy-larin-firstbridge) [reply apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/peer/PeerLoaderThread.java:55]: It seems to me that doesn't matter here.
Most of variables are initialized inside PeersService -> public void init() {......) and later are used as 'read-only' fields.

**Pull request merged**